monster-cards/src/widgets/card-details/createCardDescription.ts

```diff
@@ -1,175 +1,133 @@
 import createRenderableChildrenMixin from 'dojo-widgets/mixins/createRenderableChildrenMixin';
 import createRenderMixin, { RenderMixin, RenderMixinState, RenderMixinOptions } from 'dojo-widgets/mixins/createRenderMixin';
-import createStatefulChildrenMixin, { StatefulChildrenState, StatefulChildren, CreateChildrenResults, CreateChildrenResultsItem } from 'dojo-widgets/mixins/createStatefulChildrenMixin';
 import createCssTransitionMixin from 'dojo-widgets/mixins/createCssTransitionMixin';
 import createWidget from 'dojo-widgets/createWidget';
 import createIconLink from '../common/createIconLink';
-import { Child } from 'dojo-widgets/mixins/interfaces';
 import { h, VNode } from 'maquette';
-import WeakMap from 'dojo-shim/WeakMap';
+import Map from 'dojo-shim/Map';
 
 export type MilestoneCardDetails = {
 	name: string;
 	tagline: string;
 	description: string;
 	imageClass: string;
 	favouriteCount: number;
-	id: string;
+	cardId: string;
 }
 
-export type CardDescriptionState = RenderMixinState & StatefulChildrenState & {
-	name?: string;
-	tagline?: string;
-	description?: string;
-	imageClass?: string;
-	favouriteCount?: number;
-	id?: string;
-};
+export type CardDescriptionState = RenderMixinState & MilestoneCardDetails;
 
 type CardDescriptionOptions = RenderMixinOptions<CardDescriptionState>;
 
-type CardDescription = RenderMixin<CardDescriptionState> & StatefulChildren<Child>;
+type CardDescription = RenderMixin<CardDescriptionState>;
 
 export type CardDescriptionItem = RenderMixin<CardDescriptionState>;
 
-interface CardDescriptionChildren<C extends Child> extends CreateChildrenResults<C> {
-	name: CreateChildrenResultsItem<C>;
-	tagline: CreateChildrenResultsItem<C>;
-	description: CreateChildrenResultsItem<C>;
-	cardImage: CreateChildrenResultsItem<C>;
-	favouriteCount: CreateChildrenResultsItem<C>;
-}
+const favouriteHref = '/api/favourite/';
```

Comment:
Let's just remove this, it is not going to be a `href` from the add to favs (action of click event most likely)